sqlitex/pool_test.go

```diff
@@ -243,3 +247,58 @@ func TestPoolPutMatch(t *testing.T) {
 		dbpool1.Put(c)
 	}()
 }
+
+func TestPoolOpenInit(t *testing.T) {
+	t.Run("ok", func(t *testing.T) {
```

Comment:
Per my other comment, probably `Ok`.

Reply:
golang doesn't rename the tests other than replacing invalid characters like spaces with `_` and joining name segments with `/`.

For example,

```
$ go test ./sqlitex -v -run OpenInit
=== RUN   TestPoolOpenInit
=== RUN   TestPoolOpenInit/ok
=== RUN   TestPoolOpenInit/invalid_initScript
=== RUN   TestPoolOpenInit/interrupted
--- PASS: TestPoolOpenInit (0.00s)
    --- PASS: TestPoolOpenInit/ok (0.00s)
    --- PASS: TestPoolOpenInit/invalid_initScript (0.00s)
    --- PASS: TestPoolOpenInit/interrupted (0.00s)
PASS
ok  	crawshaw.io/sqlite/sqlitex	0.128s
```